### PR TITLE
Add EPUB

### PR DESCRIPTION
**Pull request by julian-cable**

### CODE REVIEW

**Review by daobrien — COMMENTED** [review]
Just a comment.

File: en-US/E.xml [hunk 1]
```diff
@@ -104,6 +104,18 @@
 
 		 </listitem>
 
+		 </varlistentry>
+		<varlistentry id="epub">
+		 <term>EPUB</term>
+			<listitem>
+			 <para>
+				 Use unhyphenated, and without any variations in styling or capitalization, to refer to this publishing format.
```
[the remark this answers is not in the document]
Comment (julian-cable) [reply en-US/E.xml:112]: We could cite the W3C page at https://www.w3.org/publishing/epub3/, although the link would need to be updated when EPUB3.x is no longer the current standard.

File: en-US/E.xml [hunk 2]
```diff
@@ -104,6 +104,22 @@
 
 		 </listitem>
 
+		 </varlistentry>
+		<varlistentry id="epub">
+		 <term>EPUB</term>
+			<listitem>
+			 <para>
+				Write the term unhyphenated, and without any variations in styling or capitalization, to refer to this publishing format.
```
Comment (daobrien) [inline en-US/E.xml:112]:
```suggestion
				Write as shown, without any variations in styling or capitalization, to refer to this publishing format.
```

Comment (daobrien) [reply en-US/E.xml:112]: That's what I meant about standardising on a format. Thanks.

File: en-US/E.xml [hunk 3]
```diff
@@ -104,6 +104,22 @@
...
+				 Do not use EPUB as a noun to refer to an EPUB file. 
+				 For example, instead of writing "in an EPUB", write "in an EPUB file". 
+			 </para>
+			 <para>
+			 EPUB is part of the w3.org publishing standards: 
+			 <ulink url="https://www.w3.org/publishing/epub3/">https://www.w3.org/publishing/epub3/</ulink>
+			 </para>
```
Comment (daobrien) [inline en-US/E.xml:119]: Yes, the link will eventually need updating but it points to a major version, and based on history it'll be a long time before we see v.4. I'm ok with it.

**Review by daobrien — APPROVED** [review]

**Pull request merged**